Sources/NoCodes/NoCodesViewController.swift

```diff
@@ -102,8 +104,15 @@ final class NoCodesViewController: UIViewController {
     
     Task {
       do {
-        let screen: NoCodes.Screen = try await noCodesService.loadScreen(with: screenId)
-        
+        let screen: NoCodes.Screen;
+        if let screenId = screenId {
+          screen = try await noCodesService.loadScreen(with: screenId)
+        } else if let contextKey = contextKey {
+          screen = try await noCodesService.loadScreen(withContextKey: contextKey)
+        } else {
+          throw NSError(domain: "no_screen_id_or_context_key", code: 0)
```

Comment:
We are using a different type of error in other cases. Can we use the same one here?